Scripts/RemediationScripts/Remediate-TransparentDataEncryptionForSQLDatabases.ps1

```diff
@@ -514,118 +611,130 @@ function Disable-TransparentDataEncryptionForSQLDatabases
-    $backupFolderPath = "$([Environment]::GetFolderPath('LocalApplicationData'))\AzTS\Remediation\Subscriptions\$($context.Subscription.SubscriptionId.replace('-','_'))\$($(Get-Date).ToString('yyyyMMddhhmm'))\EnableTDEForSQLServerDatabases"
+    Write-Host "Found $($totalSQLServers) SQL Server(s)." -ForegroundColor $([Constants]::MessageType.Update)
+    $backupFolderPath = "$([Environment]::GetFolderPath('LocalApplicationData'))\AzTS\Remediation\Subscriptions\$($context.Subscription.SubscriptionId.replace('-','_'))\$($(Get-Date).ToString('yyyyMMddhhmm'))\EnableTDEForSQLServers"
 
     if (-not (Test-Path -Path $backupFolderPath))
     {
         New-Item -ItemType Directory -Path $backupFolderPath | Out-Null
     }
  
-    Write-Host "Transparent Data Encryption (TDE) will be disabled for SQL Server databases." -ForegroundColor $([Constants]::MessageType.Warning)
+    Write-Host "Transparent Data Encryption (TDE) will be disabled for SQL Servers." -ForegroundColor $([Constants]::MessageType.Warning)
     if (-not $Force)
     {
-        Write-Host "Do you want to disable Transparent Data Encryption (TDE) for all SQL Server databases? " -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
+        Write-Host "Do you want to disable Transparent Data Encryption (TDE) for all SQL Servers? " -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
             
         $userInput = Read-Host -Prompt "(Y|N)"
 
         if($userInput -ne "Y")
         {
-            Write-Host "Transparent Data Encryption (TDE) will not be disabled for SQL Server databases. Exiting..." -ForegroundColor $([Constants]::MessageType.Update)
+            Write-Host "Transparent Data Encryption (TDE) will not be disabled for SQL Servers. Exiting..." -ForegroundColor $([Constants]::MessageType.Update)
             break
         }
     }
     else
     {
-        Write-Host "'Force' flag is provided. Transparent Data Encryption (TDE) will be disabled for SQL Server databases." -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
+        Write-Host "'Force' flag is provided. Transparent Data Encryption (TDE) will be disabled for SQL Servers." -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
     }
 
     Write-Host $([Constants]::DoubleDashLine)
-    Write-Host "[Step 3 of 3] Disabling Transparent Data Encryption (TDE) for SQL Server databases..." -ForegroundColor $([Constants]::MessageType.Warning)
+    Write-Host "[Step 3 of 3] Disabling Transparent Data Encryption (TDE) for SQL Servers..." -ForegroundColor $([Constants]::MessageType.Warning)
 
-    # Includes SQL Server databases, to which, previously made changes were successfully rolled back.
-    $sqlServersRolledBack = @()
+    $databaseList = @()
+    $rollbackSummary = @()
+    $totalRolledbackSQLServers = 0
+    try
+    {
+        $validSQLServerDetails | ForEach-Object{
+            $databaseDetail = $_
+            $serverName = $_.ServerName
```

Comment:
What file is being used for the rollback? The output of `DryRun` or the one containing the remediation summary.

Reply:
RemediationSummary is being used while performing rollback operation.